Makefile

```
@@ -55,76 +53,35 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
-.PHONY: test
-test: manifests generate fmt vet envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
+.PHONY: lint
+lint: fmt vet ## All-in-one linting
+	@echo 'Check for uncommitted changes ...'
+	git diff --exit-code
 
 ##@ Build
 
 .PHONY: build
-build: generate fmt vet ## Build manager binary.
-	go build -o bin/manager main.go
+build: export CGO_ENABLED = 0
+build: generate fmt vet  ## Build manager binary.
+build:
+	@echo "GOOS=$$(go env GOOS) GOARCH=$$(go env GOARCH)"
+	go build -o $(BIN_FILENAME)
 
 .PHONY: run
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
 .PHONY: docker-build
-docker-build: test ## Build docker image with the manager.
+docker-build:
+	env GOOS=$(DOCKER_IMAGE_GOOS) GOARCH=$(DOCKER_IMAGE_GOARCH) \
+		go build -o ${BIN_FILENAME}
```

We need to statically link this otherwise the container won't work when built on a machine with glibc

```suggestion
	env CGO_ENABLED=0 GOOS=$(DOCKER_IMAGE_GOOS) GOARCH=$(DOCKER_IMAGE_GOARCH) \
		go build -o ${BIN_FILENAME}
```